pkg/metrics/metrics_exporter.go

```diff
@@ -24,15 +24,19 @@ var (
 	)
 )
 
-func (h *Healer) metricsHandler() http.Handler {
```

Comment:
As this has its own package now, you can simplify the names, just `Handler`, for example. That way the user of this package will write `metrics.Handler` which reads nicely.

Reply:
nice :+1: